tests-lit/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_custom_target(tests-lit)
```

Comment:
What does it do?

Reply:
It is a target that simply executes the custom command above. Do you see anything wrong with this approach?

Reply:
Not necessarily wrong, but looks a bit suspicious to me, though it's just a gut feeling. I think it's fine.

Reply:
This has been changed.